hq/app/schedule/IamNotifications.scala

```diff
@@ -21,16 +21,16 @@ object IamNotifications extends Logging {
     }
   }
 
-  def createIamAuditUsers(users: Seq[VulnerableUser], account: AwsAccount): Seq[IamAuditUser] = {
+  private def createIamAuditUsers(users: Seq[VulnerableUser], account: AwsAccount): Seq[IamAuditUser] = {
     users.map { user =>
       IamAuditUser(
-        Dynamo.createId(account, user.username), account.name, user.username,
+        DynamoAlerts.createId(account, user.username), account.name, user.username,
         List(IamAuditAlert(VulnerableCredential, DateTime.now, createDeadlineIfMissing(user.disableDeadline)))
       )
     }
   }
 
-  def createWarningAndFinalNotification(tg: IAMAlertTargetGroup, awsAccount: AwsAccount, users: Seq[IamAuditUser]): IamNotification = {
+  private def createWarningAndFinalNotification(tg: IAMAlertTargetGroup, awsAccount: AwsAccount, users: Seq[IamAuditUser]): IamNotification = {
```

Comment:
We test this function in `test/schedule/SendTestNotifications.scala`.

```suggestion
  def createWarningAndFinalNotification(tg: IAMAlertTargetGroup, awsAccount: AwsAccount, users: Seq[IamAuditUser]): IamNotification = {
```

Reply:
Good spot, I noticed too after the TC build failed. It came from the rebase of main, so I did the same thing as your suggestion for now.